src/webots/nodes/WbConnector.cpp

```diff
@@ -373,9 +373,9 @@ void WbConnector::snapOrigins(WbConnector *other) {
       h[i] /= 2.0;
   }
 
-// gcc 12.1.0 is raising a false positive warning here about dangling pointers
+  // gcc version 12.1.0 and 12.2.0 is raising a false positive warning here about dangling pointers
```

Comment:
```suggestion
// gcc version 12.1.0 and 12.2.0 is raising a false positive warning here about dangling pointers
```

Not sure about the added spaces